invenio_banners/services/schemas.py

```diff
@@ -11,13 +11,13 @@
 
 from invenio_records_resources.services.records.schema import BaseRecordSchema
 from marshmallow import fields, pre_load
-from marshmallow_utils.fields import TZDateTime
+from marshmallow_utils.fields import SanitizedHTML, TZDateTime
 
 
 class BannerSchema(BaseRecordSchema):
     """Schema for banners."""
 
-    message = fields.String(required=True)
+    message = SanitizedHTML(required=True)
```

Comment:
Thanks for the change, it would be nice to add a test with HTML ☺️

Reply:
I agree that's good to have, I added one :)